contributing/diversity/governance.rst

```diff
@@ -0,0 +1,110 @@
+Diversity Initiative Governance
+===============================
+
+Membership
+----------
+
+Membership of Symfony’s Diversity Initiative is open to any member of the
+Symfony community; to avoid the risk of elitism or meritocracy, no requirement
+is needed to be involved. All members, at any time, are invited to put forward
+ideas and suggestions as a proposal for an actionable item.
+
+Guidance
+--------
+
+The project leader, Fabien Potencier, is responsible for publicly appointing
+five (5) key players of the initiative to provide guidance and drive it forward,
+but also retains the right to revoke any of the key players at any time. These
+guiding key players should:
```

Comment:
Should we require a certain amount of diversity for the members of this group?

Reply:
That would be nice to have and I think the initiative leans in that direction already - but requiring that would mean having to quantify it somehow... How would we measure that diversity, and what or how many minorities should we enforce having?

Reply:
I would indeed not introduce "quotas".